inc/saiport.h

```diff
@@ -478,6 +486,15 @@ typedef enum _sai_port_attr_t
     SAI_PORT_ATTR_ADVERTISED_FEC_MODE,
 
     /**
+     * @brief Advertised port FEC AN request bit.  True means FEC is requested for 10/40/100G
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_ADVERTISED_FEC_MODE_REQUEST,
```

Comment:
@lguohan - what is SAI's position between 25G IEEE & Consortium?  I noticed that there is a port attribute that mentions consortium but I don't see an explicit definition.

CC @mikelazar